pkg/controllers/federatedresourcequota/federated_resource_quota_enforcement_controller.go

```diff
@@ -141,10 +173,27 @@ func (c *QuotaEnforcementController) SetupWithManager(mgr controllerruntime.Mana
 			return false
 		},
 	})
-	return controllerruntime.NewControllerManagedBy(mgr).
+
+	resyncEvent := make(chan event.GenericEvent)
+	if c.ResyncPeriod.Duration > 0 {
+		go func() {
+			ticker := time.NewTicker(c.ResyncPeriod.Duration)
+			defer ticker.Stop()
+			for {
+				<-ticker.C
+				resyncEvent <- event.GenericEvent{}
+			}
+		}()
+	}
```

Comment:
It deserves to have a comment to explain the blank event.
```suggestion
	// Creates a resync event channel to trigger periodic synchronization.
	// If a resync period is configured (>0):
	// - Starts a goroutine that fires a blank event at each resync interval
	// - Uses time.Ticker for precise periodic scheduling
	// The empty GenericEvent acts as a signal to force reconciliation.
	resyncEvent := make(chan event.GenericEvent)
	if c.ResyncPeriod.Duration > 0 {
		go func() {
			ticker := time.NewTicker(c.ResyncPeriod.Duration)
			defer ticker.Stop()
			for {
				<-ticker.C
				resyncEvent <- event.GenericEvent{}
			}
		}()
	}
```

Reply:
By the way, can we start this goroutine by [a Start()](https://github.com/karmada-io/karmada/blob/276cb0d3d7386ebef85521b37aa19f5322fde812/vendor/sigs.k8s.io/controller-runtime/pkg/manager/manager.go#L297)?